### ISSUE TITLE
Color contrast does not meet minimum requirement - uids.brand.uiowa.edu

### ISSUE THREAD

**Issue by briand44** [uiowa/uids#1041]
# Motivation

Siteimprove is reporting "Color contrast does not meet minimum requirement". This is a WCAG 2.1 AA compliance issue.

# Siteimprove info

**Technical doc**
https://alfa.siteimprove.com/rules/sia-r69

Siteimprove link to existing issue
- https://my2.siteimprove.com/Inspector/997947/A11y/Page?pageId=69552798044&impmd=264ED37B65FA6186129334A508391970&conf=a+aa#/sia-r69/failed/RWFNa09KVUs=;ODA2ZTU1ZDQ=
- https://my2.siteimprove.com/Inspector/997947/A11y/Page?pageId=69552922806&impmd=264ED37B65FA6186129334A508391970&conf=a+aa#/sia-r69/failed/RWFNa09KVUs=;MmQxZmFkMDE=

Example
- https://uids.brand.uiowa.edu/?path=%2Fstory%2Fnot-implemented-slider--slider
- https://uids.brand.uiowa.edu/?path=/story/not-implemented-tabs--tabs

**Color contrast does not meet minimum requirement**
Issue
1.4.3: Contrast (Minimum)
There is insufficient color contrast between the text and background on this page to meet the minimum contrast requirement.

**Comment by briand44** [comment]
## Grooming
 
**Quadrant:** 

Complicated

 **Discussion:**
 - plan to update, may be fixed by that
 - if not, check style rules and config values
 - Could we just drop the yellow branding for more of a default?
 - looks like applied to individual elements themselves
 - https://github.com/uiowa/uids/blob/4.x/.storybook/uidsTheme.js

**Proposed solution or definition of done:** 
- ensure color contrast is meeting accessibility compliance
- verify siteimprove is happy
